pretext/ExtraTopics/Glossary.ptx

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-glossary">
+  <title>Glossary</title>
+
+  
+  <!-- BOILERPLATE STUFF -->
+  <glossary sorted="False">
+    <gi>
+      <title>activecode</title>
+      <p>A unique interpreter environment that allows Python to be executed from within a web browser.</p>
+    </gi>
+    <gi>
+      <title>algorithm</title>
+      <p>A general step by step process for solving a problem.</p>
+    </gi>
+    <gi>
+      <title>bug</title>
+      <p>An error in a program.</p>
+    </gi>
+    <gi>
+      <title>byte code</title>
+      <p>An intermediate language between source code and object code. Many
+                            modern languages first compile source code into byte code and then
+                            interpret the byte code with a program called a <em>virtual machine</em>.</p>
```

Comment:
I really don't like this explanation. Esp because python doesn't need to be compiled to bytecode.